uuid_test.go

```diff
@@ -281,6 +284,56 @@ func TestTimestampFromV7(t *testing.T) {
 	}
 }
 
+func TestTimestampMinMaxV167(t *testing.T) {
```

Comment:
```suggestion
func TestMinMaxTimestamps(t *testing.T) {
```
Timestamps are only valid for V1, V6 and V7 values and mashing the 3 version together makes for a clunky name IMO.